scripts/build.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
```

Comment:
Use #!, not just #, for the shebang. Also, consider using `#!/bin/sh` instead of `bash` to eliminate the dependency on it and make it more portable across POSIX-like operating systems.